app/src/js/actions/index.js

```diff
@@ -781,8 +781,6 @@ export const deleteRule = (ruleName) => ({
 
 export const enableRule = (payload) => {
   const rule = clonedeep(payload);
-  delete rule['queriedAt'];
```

Comment:
This extra property came from the reducer and seems unnecessary, so I prevented it from being added there and thus we won't have to remove it

see https://github.com/nasa/cumulus-dashboard/pull/611/files#diff-c33e901d446f106f94633d34f927341aL65